src/components/workspace-utils.js

```diff
@@ -49,7 +49,7 @@ export const WorkspaceSelector = ({ workspaces, value, onChange, ...props }) =>
     value,
     onChange: ({ value }) => onChange(value),
     options: _.flow(
-      _.sortBy('workspace.name'),
+      _.sortBy([ws => { return (ws.workspace.name.toLowerCase()) }]),
```

Comment:
nit: with short, single expression functions, it's ok to use the brace-less shorthand. Also, the array isn't needed unless there are multiple sorts.
```suggestion
      _.sortBy(ws => ws.workspace.name.toLowerCase()),
```